cal/cal/static/css/calendaranalytics.css

```diff
@@ -10,22 +10,41 @@ h1, h2, h3, h4, h5, h6, a, p {
   color: #000; }
 
 .jumbotron h1, .jumbotron .h1 {
-  color: #000; }
+  color: #000;
```

Comment:
Copied and pasted this code into the scss, thanks for that. What command should I reference in cal/README.md?